### PR TITLE
Document feature flags in readme

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Follow up to #190, this documents our feature flags in our readme. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,24 @@ fn main() {
...
+_great_ for `async-std`, and we want to provide a sneak-peek so you can try it
+out. This is what we call _"unstable"_ features. You can try out the unstable
+features by enabling the `unstable` feature in you `Cargo.toml` file:
+
+```toml
+[dependencies]
+async-std = { version = "0.99.5", features = ["unstable"] }
+```
+
+Just be careful when running these features, as they may change between
```
Comment (stjepang) [inline README.md]:
```suggestion
Just be careful when using these features, as they may change between
```

File: README.md [hunk 2]
```diff
@@ -81,6 +81,24 @@ fn main() {
...
+break unless _you_ decide to upgrade.
+
+However every now and then we come up with something that we think will work
+_great_ for `async-std`, and we want to provide a sneak-peek so you can try it
+out. This is what we call _"unstable"_ features. You can try out the unstable
+features by enabling the `unstable` feature in you `Cargo.toml` file:
+
+```toml
+[dependencies]
+async-std = { version = "0.99.5", features = ["unstable"] }
```
Comment (stjepang) [inline README.md]: In #190, I used the following syntax:

```toml
[dependencies.async-std]
version = "0.99"
features = ["unstable"]
```

Do you have a preference? I personally don't so pick whichever you like best. :)

File: README.md [hunk 3]
```diff
@@ -81,6 +81,24 @@ fn main() {
...
 
+## Features
+
+`async-std` is strongly commited to following semver. This means your code won't
+break unless _you_ decide to upgrade.
+
+However every now and then we come up with something that we think will work
+_great_ for `async-std`, and we want to provide a sneak-peek so you can try it
+out. This is what we call _"unstable"_ features. You can try out the unstable
+features by enabling the `unstable` feature in you `Cargo.toml` file:
```
Comment (stjepang) [inline README.md]:
```suggestion
features by enabling the `unstable` feature in your `Cargo.toml` file:
```

File: README.md (hunk 2 above)
Comment (yoshuawuyts) [reply README.md]:
```suggestion
[dependencies.async-std]
version = "0.99"
features = ["unstable"]
```

**Pull request merged**